guide/designing-products/personal-finance.md

```diff
@@ -226,11 +225,12 @@ Many people don’t self-cusody their long-term savings. They rely on banks and
    layout = "float-left"
 %}
 
-Investing is about putting saved money to work hoping that it increases in value, from retirement accounts to stock investing and real estate ownership. This necessarily requires the involvement of third parties, which goes beyond the current focus of this guide.
+Investing is about turning the money you have into more money over time. From individuals to governments to corporations to pension funds, the goal of investing is growth. This necessarily requires the involvement of third parties, which is beyond the focus of this guide (at least for now).
 
 </div>
 
-Since Bitcoin has dramatically increased in value, many see bitcoin itself as an investment (which is well expressed in the [HODL meme](https://knowyourmeme.com/memes/hodl){:target="_blank"}).
+Since Bitcoin continues to increase in value over time, many see the act of “HODLing” as investing. In the last ten years, an increasing number of financial advisors and institutions have come to see bitcoin as a necessary component or any sufficiently diversified portfolio.
```

Comment:
```suggestion
Since Bitcoin continues to increase in value over time, many see the act of “HODLing” as investing. In the last ten years, an increasing number of financial advisors and institutions have come to see bitcoin as a necessary component of any sufficiently diversified portfolio.
```